### PR TITLE
Fix/account tag names

### PR DESCRIPTION
**Pull request by ujjwal-cyph**
Fixes https://app.clickup.com/t/37308523/CHI-2168

### CODE REVIEW

**Comment by vbcypherock** [comment]
Task linked: [CHI-2168 Correct account tag names](https://app.clickup.com/t/37308523/CHI-2168)

File: src/controller_main.c [hunk 1]
```diff
@@ -524,10 +524,10 @@ void desktop_listener_task(lv_task_t* data)
                                      sizeof(account_name));
                     snprintf(flow_level.confirmation_screen_text,
                              sizeof(flow_level.confirmation_screen_text),
-                             "Do you want to add %s %s to %s?",
+                             "Add %s %s to %s",
                              get_coin_name(add_coin_data.derivation_path[1],
                                            add_coin_data.network_chain_id),
-                             wallet.wallet_name, account_name);
+                             account_name, wallet.wallet_name);
```
Comment (amanCypherock) [inline src/controller_main.c:530]: Update `account_name` to `account_tag`

File: src/controller_main.c [hunk 2]
```diff
@@ -566,10 +566,9 @@ void desktop_listener_task(lv_task_t* data)
                         flow_level.level_two = LEVEL_THREE_SEND_TRANSACTION_ETH;
                         snprintf(
                             flow_level.confirmation_screen_text, sizeof(flow_level.confirmation_screen_text),
-                            "Send %s with %s with %s %s", var_send_transaction_data.transaction_metadata.token_name,
-                            wallet.wallet_name,
+                            "Send %s on %s %s> from %s", var_send_transaction_data.transaction_metadata.token_name,
```
Comment (amanCypherock) [inline src/controller_main.c:569]:
```suggestion
                            "Send %s on %s %s from %s", var_send_transaction_data.transaction_metadata.token_name,
```

File: src/controller_main.c (hunk 1 above)
Comment (ujjwal-cyph) [reply src/controller_main.c]: changed

**Review by amanCypherock — APPROVED** [review]

**Pull request merged**